main.go

```diff
@@ -37,6 +37,7 @@ import (
 var (
 	configFile    = kingpin.Flag("config.file", "Path to configuration file.").Default("snmp.yml").String()
 	listenAddress = kingpin.Flag("web.listen-address", "Address to listen on for web interface and telemetry.").Default(":9116").String()
+	wrapCounters  = kingpin.Flag("wrap-large-counters", "Wrap 64-bit counters to avoid floating point rounding.").Default("true").Bool()
```

Comment:
this should be snmp. or similar

Reply:
Good idea.

Reply:
Also, I guess this should go in collector.go.